scraper/smashgg.py

```diff
@@ -36,7 +41,16 @@ def __init__(self, path):
         # JSON DUMPED FROM THE API
 
         self.event_dict = SmashGGScraper.get_event_dict(self.event_id)
-        self.group_ids = self.get_group_ids()
+
+        self.group_sets = []
+        for phase in self.included_phases:
+            self.group_sets.append(SmashGGScraper.get_group_ids_from_phase(phase))
+
+        self.group_ids = []
+        for group_set in self.group_sets:
+            for group_id in group_set:
+                self.group_ids.append(group_id)
+
         self.group_dicts = [SmashGGScraper.get_group_dict(group_id) for group_id in self.group_ids]
```

Comment:
so unfortunately for large tournaments (e.g. Genesis) this eats a pretty large amount of memory (~ 100M; not so bad, but given our current memory problems can break stuff).

probably we should only pull the relevant parts of the group_dict to alleviate the problem for now.